Components/Navbar/navbar.css

```diff
@@ -0,0 +1,187 @@
+.navbar-home {
+  padding: 0.7rem 0rem;
+  box-shadow: 0px 0px 10px 4px rgb(0 0 0 / 10%);
+}
+.navbar-wrapper {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 1rem;
+  position: sticky;
+  top: 0;
+  z-index: 5;
+  height: 2.5rem;
+}
+.navbar-left-aligned {
+  display: flex;
+  align-items: center;
+  width: 50%;
+}
+.navbar-left-aligned svg {
+  width: 1.875rem;
+  height: 1.875rem;
+}
+.sidebar-toggler {
+  display: none;
+}
+.brand-logo {
+  font-weight: bold;
+  font-size: 2rem;
+  color: var(--primary-color);
+  margin-right: 3rem;
+}
+.navbar-right-aligned {
+  display: flex;
+  align-items: center;
+  justify-content: flex-end; 
+  width: 50%;
+}
+.navbar-nav {
+  display: flex;
+  align-items: center;
+}
+.nav-item-link {
+  padding: 0.5rem 0.75rem;
+  display: block;
+  text-decoration: none;
+  font-size: 1rem;
+  color: var(--dark-secondary-text-color);
+}
+.nav-icon-link{
+  padding: 0.5rem 0.75rem;
+  display: block;
+  text-decoration: none;
+  font-size: 1rem;
+  color: var(--dark-secondary-text-color);
+}
+.nav-icon{
+  width: 1.3rem;
+  height: 1.3rem;
+  font-size: 1.1rem;
+}
+.search-bar{
+  display: flex;
+  align-items: center;
+  flex-grow: 1;
+  position: relative;
+  background-color:  #f7f7fd;
+}
+.search-bar__btn{
+  padding: 0.5rem 0.7rem;
+  background: transparent;
+  border: none;
+  font-size: 1rem;
+  color: var(--dark-secondary-text-color);
+}
+.search-bar-input{
+  padding: 0.5rem 0.7rem;
+  border: none;
+  background-color:  #f7f7fd;
+  width: 100%;
+}
+.search-bar-input:focus{
+  border-bottom: 1px solid var(--dark-secondary-text-color);
+}
+.responsive-img {
+  block-size: auto;
+  max-width: 100%;
+  width: 100%;
+  height: auto;
+}
+.hero-img-wrapper{
+  max-height: calc(100vh - 72px);
+  position: relative;
+}
+.text-overlay{
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+  height: 100%;
+  width: 100%;
+  top: 0px;
+  left: 0;
+}
+.text-overlay-title{
+  color: black;
+  letter-spacing: 2px;
+}
+
+/* sidebar to be developed */
+.sidebar-wrapper {
+  display: none;
+}
+.navbar-collapse {
+  display: block;
+}
+.sidebar-wrapper {
+  background-color: rgba(0, 0, 0, 0.562);
+  width: 100vw;
+  z-index: 3;
+  visibility: hidden;
+  opacity: 0;
+  transition: all 0.3s cubic-bezier(0.25, 0.46, 0.43, 0.96);
+}
+/* .sidebar-wrapper,
+.sidebar {
+  height: 100vh;
+    position: fixed;
+    top: 40px;
+    left: 0;
+} */
+.sidebar {
+  /* max-width: 300px;
```

Comment:
If they are not useful , you can remove commented code

Reply:
These styles are required for toggle button on landing page. Yet to be developed